src/stache/includes/components/tabs-containers.html

```diff
@@ -0,0 +1,27 @@
+<p>
+  If tabs and vertical tabs contain removable items such as lists or reports, don't create entries in the browser history.
+</p>
+
+<div class="screenshot screenshot-info"> 
+  <img src="~/assets/img/guidelines/back-button/tabcontainers-backbutton.png" alt="Tabs as containers for viewing items" />
```

Comment:
We have 2 images here, while we only have 1 image (no vertical tabs image) in the previous section. Is this correct?